.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
- [ ] OPTIONAL
Kindly keep this file and folder structure in the `.gitignore` file so as to keep your folder structure clean